docs/data-management/intro.md

```diff
@@ -0,0 +1,122 @@
+After data collection, a significant amount of post-scanner steps remain to reach the point where all data and derivatives have been processed, making sure they are BIDS compliant and ready for release.
+
+The flowchart below shows an overview of the post-scanner data workflow:
+
+``` mermaid
+flowchart TB
+    classDef imp stroke:#f00,stroke-width:2px;
+
+    subgraph PACS[PACS]
+    end
+
+    subgraph physio["Physio-recording Laptop ({{ secrets.hosts.acqknowledge | default("███") }})"]
+    end
+
+    subgraph et_pc[Eye-tracker Computer]
+    end
+
+    subgraph psychopy_pc["Stimuli presentation Laptop ({{ secrets.hosts.psychopy | default("███") }})"]
+    end
+
+    subgraph local_data[Local Storage]
```

Comment:
Is it possible to write the label of the box in bold so it stands out more?

Reply:
I tried but the markdown formating is messy with mermaid (especially when rendered through mkdocs), I'll investigate

Reply:
It works now, you should use html formatting such as `<b>bold</b> and <i>italic</i>` to render properly.